queries.sql

```diff
@@ -96,6 +96,52 @@ Select species, AVG(escape_attempts)
 FROM animals 
 where date_of_birth between '1990-01-01' AND '2000-12-31' GROUP BY species;
 
-
-
-
+-----Write queries (using JOIN) to answer the following questions:
+---What animals belong to Melody Pond?
+
+select animals.name, owners.full_name
+from animals 
+inner join owners on 
+animals.owner_id = owners.id
+where owners.full_name ='Melody Pond'
+;
```

Comment:
- [ ] @leloufadel Great work adding a query to find what animals belong to Melody Pond.However the screenshot provided doesn't showcase the same query and expected result kindly consider changing it.